File: 01-Ejercicios-Clases-2023_2/C8-09-01-23--09-08-23/ProyectosJAVA/LP2SoftRRHHController/src/pe/edu/pucp/lp2soft/rrhh/dao/EmpleadoDAO.java
package pe.edu.pucp.lp2soft.rrhh.dao;
import java.util.ArrayList;
import pe.edu.pucp.lp2soft.rrhh.model.Empleado;
public interface EmpleadoDAO {
    int insertar(Empleado empleado);
    int modificar(Empleado empleado);
    int eliminar(int idEmpleado);
    ArrayList<Empleado> listarTodos();
}

File: 04-Codigo-Clases/BackCSharp_FrontJava/CSharp/LP2Soft/LP2SoftRRHHController/DAO/EmpleadoDAO.cs
using LP2SoftRRHHModel;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LP2SoftRRHHController.DAO
{
    public interface EmpleadoDAO
    {
        int insertar(Empleado empleado);
        int modificar(Empleado empleado);
        int eliminar(int idEmpleado);
        BindingList<Empleado> listarPorNombreDNI(string nombreDNI);
        BindingList<Empleado> listarTodos();
        Empleado buscarPorId(int idEmpleado);
    }
}
